samples/python2/mser.py

```diff
@@ -26,7 +26,7 @@
         video_src = 0
 
     cam = video.create_capture(video_src)
-    mser = cv2.MSER()
+    mser = cv2.MSER_create()
```

Comment:
I have one note regarding `mser.py` example. Currently it does not work. To fix it we need to call `detectRegions` instead of `detect` in python script and add `CV_OUT` prefix like this:
```
    CV_WRAP virtual void detectRegions( InputArray image,
                                        CV_OUT std::vector<std::vector<Point> >& msers,
                                        std::vector<Rect>& bboxes ) = 0;
```
 to `MSER::detectRegions` method in `features2d.hpp`.